### PR TITLE
[PersistentStorage] Update readme and changelog after rename

### PR DESCRIPTION
**Pull request by vishweshbankwar**

### CODE REVIEW

File: src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+ [`OpenTelemetry.PersistentStorage.Abstractions`](https://www.nuget.org/packages/OpenTelemetry.PersistentStorage.Abstractions).
+ Older versions will remain at
+ [`OpenTelemetry.Extensions.PersistentStorage.Abstractions`](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage.Abstractions)
+ [#1079](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1079)
+
+  Migration:
+
+  * In code update namespaces (eg `using
+    OpenTelemetry.Extensions.PersistentStorage.Abstractions` -> `using
+    OpenTelemetry.PersistentStaorage.Abstractions`)
```
Comment (cijothomas) [inline src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md:15]:
```suggestion
    OpenTelemetry.PersistentStorage.Abstractions`)
```

File: src/OpenTelemetry.PersistentStorage.Abstractions/README.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.PersistentStorage.Abstractions.svg)](https://www.nuget.org/packages/OpenTelemetry.PersistentStorage.Abstractions)
 
 This package includes APIs which can be extended by exporter owners to implement
 persistent storage.
 
 ## Installation
 
 ```shell
-dotnet add package OpenTelemetry.Extensions.PersistentStorage.Abstractions
+dotnet add package OpenTelemetry.PersistentStorage.Abstractions
```
Comment (cijothomas) [inline src/OpenTelemetry.PersistentStorage.Abstractions/README.md:12]: prerelease flag

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+
+* Going forward the NuGet package will be
+ [`OpenTelemetry.PersistentStorage.Abstractions`](https://www.nuget.org/packages/OpenTelemetry.PersistentStorage.Abstractions).
+ Older versions will remain at
+ [`OpenTelemetry.Extensions.PersistentStorage.Abstractions`](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage.Abstractions)
+ [#1079](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1079)
+
+  Migration:
+
+  * In code update namespaces (eg `using
```
Comment (utpilla) [inline src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md:13]:
```suggestion
  * In code, update namespaces (e.g. `using
```

File: src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,18 @@
 # Changelog
 
-// TODO: update changelog
+## 1.0.0-beta.1 (Unreleased)
```
Comment (utpilla) [inline src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md:3]: Change this to have a different version from the old package?

Comment (vishweshbankwar) [reply src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md:3]: Was not sure about this. I will update

**Review by utpilla — APPROVED** [review]

**Pull request merged**